pallets/streaming/src/lib.rs

```diff
@@ -161,7 +173,17 @@ pub mod pallet {
     /// Global storage for streams
     #[pallet::storage]
     #[pallet::getter(fn get_stream)]
-    pub type Streams<T: Config> = StorageMap<_, Twox64Concat, StreamId, Stream<T>>;
+    pub type Streams<T: Config> = StorageMap<_, Blake2_128Concat, StreamId, Stream<T>, OptionQuery>;
```

Comment:
Feel we just need StorageDoubleMap: sender -> receiver -> Vec<Stream>

Reply:
This is a front-end requirement, we need to query the scent and received streams based on AccountId respectively.

Reply:
I think CJ means that we could use a StorageDoubleMap so that we could query either by the sender or receiver to save the storage?

Reply:
I understand it, CJ means that the keys of this two storage are different, but they maintain the same values, so it can be optimized by using double keys as the storage structure. I agree with this, but the requirement from the front end makes it impossible to use `StorageDoubleMap `. Take `<AssetId, AccountId> -> Balance` as an example, we must provide both keys in order to query the corresponding value, but when we query a receive stream, we don't care who is the sender.

Reply:
Yeah for sure.I think you are right about it.

Reply:
@rjman-ljm how about `StorageNMap : assetId -> sender -> receiver -> Vec`

Reply:
though I'm not sure if the frontend is easy to get received streams

Reply:
Three keys? I think it would be unfriendly for front-end to query If both sender and receive are keys.

Reply:
How about `StorageDoubleMap: AccountId -> StreamKind(Send | Receive) -> StreamId(u32)`?